vacuum_world.ipynb

```diff
@@ -341,25 +479,45 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": 21,
    "metadata": {},
    "outputs": [],
    "source": [
     "# TODO: Implement these functions for two-dimensional environment\n",
     "# Interpret-input function for the two-state environment\n",
-    "def interpret_input(percept):\n",
-    "    pass\n",
-    "\n",
-    "rules = None\n",
+    "def interpret_input(state):\n",
+    "    return state\n",
     "\n",
     "# Rule-match function for the two-state environment\n",
```

Comment:
We don't need any of this rule mechanism.  All we need is:

     def VacuumAgentProgram(percept):
        loc, status = percept
        return ('Suck' if status == 'Dirty' else
                'Right' if loc = loc_A else
                'Left')